peter-rose/short-response.md

```diff
@@ -58,4 +79,6 @@
      counter += 1;
    }
    ```
-
+<!--Answers-->
+   This causes an infitnite loop becasue counter is declared every time the loop is run.
```

Comment:
It's not **declared** every time, it's **reassigned to `1`** each time. Make this change.